python/ray/serve/api.py

```diff
@@ -617,6 +626,40 @@ def connect() -> Client:
     return Client(controller, controller_name, detached=True)
 
 
+def get_current_backend_tag() -> BackendTag:
+    """When called from within a backend, return its backend tag.
+
+    Raises:
+        RayServeException if not called from within a Ray Serve backend.
+    """
+    if _INTERNAL_REPLICA_CONTEXT is None:
+        raise RayServeException("Called `serve.get_current_backend_tag()`"
+                                "from a function that is not running in a "
+                                "Ray Serve backend on this cluster, so it "
+                                "has no backend tag yet. Please "
+                                "use client.create_backend() to run your "
+                                "backend function using Ray Serve.")
+    else:
+        return _INTERNAL_REPLICA_CONTEXT.backend_tag
+
+
+def get_current_replica_tag() -> ReplicaTag:
+    """When called from within a backend, return its replica tag.
```

Comment:
"Replica tag" is not currently a user-facing concept, so we need to explain it a bit here. Can you add a description? e.g., "uniquely identifies a single replica (process) for a backend"